book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :author, :title
+  attr_reader :rental
```

Comment:
- [ ] Kindly add your rental variable to attribute accessor,  based on the requirements `Getters and Setters for @rentals should be present`

Reply:
Well resolved ✔️